src/main/java/com/forgeessentials/core/ForgeEssentials.java

```diff
@@ -287,5 +291,15 @@ public void setDebugMode(boolean debugMode)
     {
         this.debugMode = debugMode;
     }
+    
```

Comment:
No. I should have code already in place to check if a client has the FE client - if not I'll get it in place soon.